src/components/JsonLink.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Icon } from './Icon.js';
+import Colors from './Colors.js';
+
+export default function JsonLink(props) {
+  const handleClick = (e) => {
+    e.preventDefault();
+    window.open(`data:application/json, ${encodeURIComponent(JSON.stringify(props.rawjson, null, 1))}`, '_blank');
+  };
+
+  return (
+    <div className="results--json">
+      <a
+        onClick={handleClick}
+        target="_blank"
+        className="json-link base--a"
+        rel="noopener noreferrer"
+        href
+      >
+        <span className="json-link--span">JSON</span> <Icon type="link-out" size="small" fill={Colors.blue_50} />
+      </a>
+    </div>
+  );
+}
+
+JsonLink.propTypes = {
+  rawjson: React.PropTypes.string.isRequired,
```

Comment:
I would recommend renaming the prop to just `json`, and also accepting objects (with automatic JSON stratification). But, neither are blockers.